mindmeld/cli.py

```diff
@@ -795,7 +796,7 @@ def _get_auto_annotator_config(app_path, overwrite=False, unannotate_all=False):
 
 @shared_cli.command("augment", context_settings=CONTEXT_SETTINGS)
 @click.option(
-    "--app-path",
+    "--app_path",
```

Comment:
In MindMeld cli, looks like we are using `--app-path`, maybe we should stick to that?